apps/surfer/public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="%REACT_APP_SURFER_GUIDELINES_URL%"></script>
+    <style>
+      iframe,
+      #root {
+        width: 100vw;
+        border: none;
+        overflow: hidden;
```

Comment:
Is this necessary? Better to have scroll when styles break 😄

Reply:
Without it the scrolls are always showing and it doesn't look great IMO. `overflow: hidden` on the other hand doesn't seem to impair any functionality.

Reply:
You're right, I changed it.